src/components/Text/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import useStyles from './styles';
+import classNames from 'classnames';
+
+interface TextProps {
+  tag?: string,
+  fontFamily?: string,
+  fontSize?: number | string,
+  lineHeight?: number | string,
+  fontStyle?: string,
+  fontWeight?: 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900,
+  fontVariant?: 'normal' | 'small-caps',
+  textTransform?: 'lowercase' | 'uppercase' | 'capitalize' | 'none',
+  marginTop?: string | number,
+  marginLeft?: string | number,
+  marginRight?: string | number,
+  marginBottom?: string | number,
+  className?: string,
```

Comment:
Does it make sense to include such properties like `fontSize`, `marginTop` etc, which could be added from JSS with `className` prop?

Reply:
updated!